Configuration/Geometry/README.md

```diff
@@ -94,11 +99,13 @@ Several detector combinations have been generated:
 * D68 = T21+C11+M6+I11+O5+F4 (For HGCAL study on evolution of detector)
 * D70 = T21+C13+M7+I11+O6+F6 (For HGCAL study on evolution of detector)
 * D76 = T21+C14+M9+I13+O7+F6
-* D77 = T24+C14+M9+I13+O7+F6
+* D77 = T24+C14+M9+I13+O7+F6 (Current default scenario)
```

Comment:
What do you mean by the current default scenario? 
The default scenario is D76. 

Of course, we can try to move to D77 where T24==T21 but DD4hep support.

Reply:
It cannot be. D77 is identical to D76. While D77 can run with dd4hep, D76 does not.

Reply:
I understand. I just say that the current geometry we used for relvals and short matrix test is D76.